internals/generators/container/index.js

```diff
@@ -20,10 +20,10 @@ module.exports = {
     },
   }, {
     type: 'list',
-    name: 'component',
-    message: 'Select a base component:',
-    default: 'PureComponent',
-    choices: () => ['PureComponent', 'Component'],
+    name: 'type',
+    message: 'Select the base component type:',
+    default: 'Stateless Function',
+    choices: () => ['Stateless Function', 'PureComponent', 'Component'],
```

Comment:
Yeah, that makes sense, after reading again I tend to agree, thanks @KarandikarMihir